Hydrator/HydrateWithMapTrait.php

```diff
@@ -28,7 +28,7 @@ protected function hydrateUserWithAttributesMap(
 
             $ldapValue = $ldapUserAttributes[$attr['ldap_attr']];
 
-            if (!array_key_exists('count', $ldapValue) || $ldapValue['count'] == 1) {
+            if ((!array_key_exists('count', $ldapValue) && count($ldapValue) == 1) || (array_key_exists('count', $ldapValue) && $ldapValue['count'] == 1)) {
```

Comment:
I've found this part has lots of collateral effects. For example collections with a single element won't be returned as array